PyInstaller/depend/dylib.py

```diff
@@ -92,6 +92,7 @@
     r'api-ms-win-core.*',
     r'api-ms-win-crt.*',
     r'ucrtbase\.dll',
+    r'vcruntime140\.dll',
```

Comment:
Does this match to the comment above: "Univeral C Runtime which consists of these DLLs"? Is vcruntime140 part of the Univeral C Runtime?